serverless.ts

```diff
@@ -33,6 +33,8 @@ const serverlessConfiguration: AWS = {
       DB_ENDPOINT: '${env:DB_ENDPOINT}',
       DB_NAME: '${env:DB_NAME}',
       DB_PORT: '${env:DB_PORT}',
+      WS_DOMAIN: '${env:WS_DOMAIN}',
+      WEBSOCKET_PORT: '${env:WEBSOCKET_PORT}',
```

Comment:
Does any deployment script needs to declare these variables?

Reply:
Deployment is being done manually at this stage via the `make deploy-lambdas-dev-testnet` command. It uses the variables from the local `.env` to do so.